example/src/main/scala/com/avast/server/toolkit/example/Main.scala

```diff
@@ -25,6 +27,7 @@ object Main extends CatsApp {
     } yield ()
   }
 
+  @silent("dead code") // false positive
```

Comment:
And again... we are putting something to our build to protect us but in the end it just bothers us and we have to put comments like this in the code. More and more I am against this overall, sorry.

Reply:
I think we can tolerate one or two false positives, it would still be a net benefit, because it catches real errors too